doc/releases/changelog-dev.md

```diff
@@ -325,6 +325,59 @@
   1: ────────────╰X──RY(-0.50)─┤
   ```
 
+* The :func:`~.transforms.decompose` transform now accepts a `stopping_condition` argument with graph-based decomposition enabled, which must be
```

Comment:
I'll defer to @isaacdevlugt , but this seems like a rather large changelog for a rather small change. We already have issues with changelog's getting extremely unwieldly by the end of a cycle.  Should we move all this to the function docstring instead?

Reply:
This section is literally duplicated from the docstring. Should we just cross reference the docstring here?